main.py

```diff
@@ -0,0 +1,77 @@
+import os
+from distr import (
+    beta,
+    binomial,
+    cauchy,
+    chi_squared,
+    dirichlet,
+    exponential,
+    exponential_exp1,
+    fisher_f,
+    frechet,
+    gamma,
+    geometric,
+    gumbel,
+    hypergeometric,
+    inverse_gaussian,
+    log_normal,
+    normal,
+    normal_inverse_gaussian,
+    pareto,
+    pert,
+    poisson,
+    skew_normal,
+    standard_geometric,
+    standard_normal,
+    student_t,
+    triangular,
+    unit_ball,
+    unit_circle,
+    unit_disc,
+    unit_sphere,
+    weibull,
+    zeta,
+    zipf,
+)
+
+
+if __name__ == "__main__":
+    out = "diagrams"
+    ext = "svg"
+    if not os.path.exists(out):
+        os.makedirs(out)
+    for distr in (
+        # beta,
+        # binomial,
```

Comment:
These shouldn't all be commented out?

Reply:
I frequently comment these in and out to run only the scripts which I have just changed. For the final commit I will make sure they are all present.